src/prowjobsscraper/equinix.py

```diff
@@ -17,9 +41,15 @@ class EquinixExtractor:
     _METADATA_PATH_TEMPLATE: Final[
         str
     ] = "{}/artifacts/{}/baremetalds-packet-gather-metadata/artifacts/equinix-metadata.json"
+    _EQUINIX_ENDPOINT_HEADER: Final[str] = "X-Auth-Token"
+    _EQUINIX_METAL_ENDPOINT: Final[
+        str
+    ] = "https://api.equinix.com/metal/v1/projects/{}/usages"
 
-    def __init__(self, client: storage.Client):
+    def __init__(self, client: storage.Client, project_id: str, project_token: str):
```

Comment:
I think the usages should live in its own file/class `equinix_usages.py`/`EquinixUsagesExtractor`, and this current file/class should be renamed `equinix_metadata.py`/`EquinixMetadataExtractor` as they don't really on the same data source